sdk/cosmos/azure-cosmos/azure/cosmos/_change_feed/change_feed_utils.py

```diff
@@ -0,0 +1,155 @@
+        3. continuation
+        4. max_item_count
+    :type args: Tuple[Any, ...]
+    :param kwargs: Keyword arguments
+    :type kwargs: dict[str, Any]
+    """
+    if len(args) > 4:
+        raise ValueError(f"Too many arguments. Expected: less than 4, but given: {len(args)}")
+
+    if len(args) > 0:
+        key_and_types = [
+            ('partition_key_range_id', str),
+            ('is_start_from_beginning', bool),
+            ('continuation', str),
+            ('max_item_count', int),
+        ]
+        for i, value in enumerate(args):
+            key, expected_type = key_and_types[i]
+
+            if key in kwargs:
+                raise ValueError(f"'{key}' is in both positional and keyword argument list. Please remove one of them.")
+
+            if not isinstance(value, expected_type):
+                raise TypeError(f"'{value}' is not of type '{expected_type.__name__}'")
+            kwargs[key] = value
+
+def validate_kwargs(
+        kwargs: Dict[str, Any]
+    ) -> None:
+    """Validate keyword arguments(kwargs).
+    The values of keyword arguments must match the expect type and conditions. If the conditions do not match,
+    errors will be raised with the error messages and possible ways to correct the errors.
+
+    :param kwargs: Keyword arguments to verify for query_items_change_feed API
+    :keyword change_feed_mode: Must be one of the values in the Enum, 'ChangeFeedMode'.
+        If the value was 'ALL_VERSIONS_AND_DELETES', the following keywords cannot be used:
+            - 'partition_key_range_id'
+            - 'is_start_from_beginning'
+            - 'start_time'
+    :type change_feed_mode: ChangeFeedState
+    :keyword partition_key_range_id: Deprecated Warning.
+    :type partition_key_range_id: str
+    :keyword is_start_from_beginning: Deprecated Warning. Cannot be used with 'start_time'.
+    :type is_start_from_beginning: bool
+    :keyword start_time: Must be in supported types.
+    :type start_time: Union[~datetime.datetime, Literal["Now", "Beginning"]]
+
+
+
+    :type kwargs: dict[str, Any]
+    """
+    # Filter items with value None
+    kwargs = {key: value for key, value in kwargs.items() if value is not None}
+
+    # Validate the keyword arguments
+    if "change_feed_mode" in kwargs:
+        change_feed_mode = kwargs["change_feed_mode"]
+        if change_feed_mode not in ChangeFeedMode:
+            raise ValueError(
+                f"Invalid change_feed_mode was used: '{kwargs['change_feed_mode']}'."
+                f" Supported 'change_feed_modes' are {ChangeFeedMode.to_string()}.")
+
+        if change_feed_mode == ChangeFeedMode.ALL_VERSIONS_AND_DELETES:
+            if "partition_key_range_id" in kwargs:
+                raise ValueError(
+                    f"'{ChangeFeedMode.ALL_VERSIONS_AND_DELETES}' mode is not supported with 'partition_key_range_id'."
+                    f" Please use 'feed_range' instead.")
+            if "is_start_from_beginning" in kwargs:
```

Comment:
Same comments as above:

if the changeFeedMode is allVersionsAndDeletes, start_time can still be used when the value is "Now"? and also I the is_start_from_beginning is false?